examples/ipython/dop.ipynb

```diff
@@ -464,7 +464,7 @@
     {
      "data": {
       "text/latex": [
-       "$\\displaystyle \\sqrt{\\left(\\operatorname{F^{\\phi}}\\right)^{2}{\\left(r,\\theta,\\phi \\right)} + \\left(\\operatorname{F^{r}}\\right)^{2}{\\left(r,\\theta,\\phi \\right)} + \\left(\\operatorname{F^{\\theta}}\\right)^{2}{\\left(r,\\theta,\\phi \\right)}}$"
```

Comment:
This one is probably a downgrade, but the fact it demonstrates flaws in our printer is a good thing, rather than just exercising the sympy one.